### ISSUE TITLE
Installed JB version of the app but there is no setting or icon to open HUD.

### ISSUE THREAD

**Issue by Jook3r** [Lessica/TrollSpeed#32]
I installed the deb file for arm64 in my device but there is no setting in Settings app assuming it shows settings instead of trollspeed icon but there is no icon either. The package is installed though. Is there any dependency for it, I simply downloaded the deb file and installed using filza. I already had trollstore version but I had been looking for a tweak which does the same and just found out there is already a deb provided by you. Thanks for such a simple and awesome app. Been using trollstore version since long before dopamine. Basically I wish to have the deb so whenever I restart and rejailbreak everything starts instead of running trollspeed seperately each time it happens. I am sorry if this is not the place to ask for such help. You can delete the post if that is the case.

**Expected behavior**
Some app or settings to enable it. 

**Smartphone:**
 - Device: iPhone 13 pro max
 - OS: iOS 16.1.2

**Comment by Lessica** [comment]
I found this problem few months ago. And I tried several days with no conclusion.

I guess `uikittools-ng` and `ldid` have some bugs and cause the same executable with different fake signature? different `CFBundleVersion` or `CFBundleShortVersionString`? failed to register:

```txt
Error: Unable to register /private/preboot/4B206E78C962519095F04705E0D39B78D527FF58FB58DE1F33FAAF076C1B76725274F6B00F306BA26B196A4761BDB16C/jb-9zI59U/procursus/Applications/TrollSpeed.app
```

We need your help: @opa334

**Comment by Lessica** [comment]
Fixed in [v1.11.11](https://github.com/Lessica/TrollSpeed/releases/tag/v1.11.11).
